react/features/gifs/functions.ts

```diff
@@ -57,44 +59,44 @@ export function getGifUrl(gif) {
  * @param {string} url - GIF url.
  * @returns {string}
  */
-export function formatGifUrlMessage(url) {
+export function formatGifUrlMessage(url: string) {
     return `${GIF_PREFIX}${url}]`;
 }
 
 /**
  * Get the Giphy API Key from config.
  *
- * @param {Object} state - Redux state.
+ * @param {IState} state - Redux state.
  * @returns {string}
  */
-export function getGifAPIKey(state) {
-    return state['features/base/config']?.giphy?.sdkKey;
+export function getGifAPIKey(state: IState) {
+    return state['features/base/config']?.giphy?.sdkKey ?? '';
 }
 
 /**
  * Returns whether or not the feature is enabled.
  *
- * @param {Object} state - Redux state.
+ * @param {IState} state - Redux state.
  * @returns {boolean}
  */
-export function isGifEnabled(state) {
+export function isGifEnabled(state: IState) {
     const { disableThirdPartyRequests } = state['features/base/config'];
     const { giphy } = state['features/base/config'];
 
     if (navigator.product === 'ReactNative' && window.JITSI_MEET_LITE_SDK) {
         return false;
     }
 
-    return !disableThirdPartyRequests && giphy?.enabled && Boolean(giphy?.sdkKey);
+    return Boolean(!disableThirdPartyRequests && giphy?.enabled && Boolean(giphy?.sdkKey));
```

Comment:
Why the extra Boolean constructor?

Reply:
Because `giphy?.enabled` could be undefined the function could return `undefined` and we want the function return type to be `boolean`